data/blog/from-native-to-js.md

```diff
@@ -0,0 +1,103 @@
+---
+title: From Native to JS in Electron
+author: codebytere
+date: '2018-03-19'
+---
+
+How do Electron's features written in C++ or Objective-C get to JavaScript so they're available to an end-user?
+
+---
+
+## Background
+
+[Electron](https://electronjs.org) is a JavaScript platform whose primary purpose is to lower the barrier to entry for developers to build robust desktop apps. However, at its core, Electron doesn't remove the need for platform-specific functionality to be written in a given system language.
+
+In reality, Electron handles the native code for you so that you can focus on a single JavaScript API.
+
+How does that work, though? How do Electron's features written in C++ or Objective-C get to JavaScript so they're available to an end-user?
+
+To trace this pathway, start with the [`app` module](https://electronjs.org/docs/api/app).
```

Comment:
Not sure if this is the intended tone of this post but it felt like it fitted in with the "walk through" style of the post

```suggestion
To trace this pathway, let's start with the [`app` module](https://electronjs.org/docs/api/app).
```